src/mixins/document-mixin.ts

```diff
@@ -2,11 +2,22 @@ import { AxiosResponse } from 'axios'
 import axios from '@/utils/axios-auth'
 import { Component, Vue } from 'vue-property-decorator'
 import { DocumentUpload } from '@/interfaces'
+import { PdfPageSize } from '@/enums'
+import pdfjsLib from 'pdfjs-dist/build/pdf'
+pdfjsLib.GlobalWorkerOptions.workerSrc = 'https://cdn.jsdelivr.net/npm/pdfjs-dist@2.0.943/build/pdf.worker.min.js'
 
 @Component({})
 export default class DocumentMixin extends Vue {
   readonly UPLOAD_FAILED_MESSAGE = 'An error occurred while uploading.  Please try again.'
   readonly MAX_FILE_SIZE = 10 * 1024 // 10 MB in KB
+  readonly pageSizeInfo = {
+    'LETTER': {
+      'pointsPerInch': 72,
+      'width': 8.5,
+      'height': 11,
+      'validationErrorMsg': 'Document must be set to fit onto 8 ½” x 11” letter-size paper'
```

Comment:
I haven't seen this style for paper size. I suggest `8.5" x 11"` instead.

Reply:
i talked to Tracey.  She suggested this validation error message.  It is probably pretty hard to read though.

Reply:
Please ask Tracey again, mentioning that I'm pushing back a bit. (Just according to what I've seen elsewhere.)

Reply:
Talked to Tracey and she's ok with `8.5" x 11"`.  Updated validation error message as well as upload instructions in `UploadRules` and `UploadMemorandum` components to use this.

Reply:
Looks good